grid_map_core/CMakeLists.txt

```diff
@@ -127,6 +130,4 @@ endif()
 
 ament_export_targets(export_${PROJECT_NAME} HAS_LIBRARY_TARGET)
 ament_export_dependencies(Eigen3)
-ament_package(CONFIG_EXTRAS
-  cmake/${PROJECT_NAME}-extras.cmake
```

Comment:
The cmake extras should not be needed, since the public `target_compile_definitions` will be propagated to all dependencies.